metadata/page/page.future-resources-.json

```diff
@@ -17,7 +17,7 @@
       "$definition": true,
       "_id": "page.future-resources---content--6",
       "_type": "content",
-      "html": "## Consider organising the repair work yourself ##\r\n\r\nIf the problem is relatively small, you could arrange for the repairs to be done and take the cost out of your rent. However, it's important to keep your landlord informed, and follow a step-by-step process. You must not stop paying rent without notice. \r\n\r\n[Find out more about organising repairs yourself](https://england.shelter.org.uk/housing_advice/repairs/doing_the_repairs_if_your_landlord_wont) on the Shelter website. ",
+      "html": "## Consider organising the repair work yourself ##\r\n\r\nIf the problem is relatively small, you could arrange for the repairs to be done and take the cost out of your rent. However, it's important to keep your landlord informed, and follow a step-by-step process. You must not stop paying rent without notice. \r\n\r\n[Find out more about organising repairs yourself](https://england.shelter.org.uk/housing_advice/repairs/doing_the_repairs_if_your_landlord_wont) on the Shelter website. \r\n\r\n## Get free legal help if you are at risk of losing your home\r\n\r\nGovernment funded legal advice and representation (legal aid) is available to people facing the loss of their home. You must have written evidence (for example a letter from your landlord saying they are going to evict you) to access support.\r\n\r\nFind out more about [legal aid for possession proceedings](https://www.gov.uk/guidance/legal-aid-for-possession-proceedings).",
```

Comment:
In case you weren't aware, the https://www.gov.uk/guidance/legal-aid-for-possession-proceedings link doesn't work.